### PR TITLE
Add qDebug Information

### PR DESCRIPTION
**Pull request by sainho93**

### CODE REVIEW

File: src/XML/sourcereader.cpp [hunk 1]
```diff
@@ -48,23 +51,26 @@ bool SourceReader::read(QIODevice *device)
             xml.raiseError(QObject::tr("The file is not an version 0.8 file."));
         }
     }
-
+    qDebug("Return SourceReader::read");
```
Comment (chraibi) [inline src/XML/sourcereader.cpp]: do you mean `Leave`?

Comment (sainho93) [reply src/XML/sourcereader.cpp]: yes, it should be "Leave"

Comment (sainho93) [reply src/XML/sourcereader.cpp]: It's be Corrected in later PR

**Comment by chraibi** [comment]
@zeroset can this PR be accepted?

**Comment by zeroset** [comment]
It is already merged?!

**Comment by sainho93** [comment]
I saw this PR is approved, then merged it. All "Return" in qDebug I've corrected.Is there still something wrong in the PR?

**Comment by chraibi** [comment]
It's ok! Thanks @sainho93 and @zeroset

**Pull request merged**